test/deploy-pipeline-lite.sh

```diff
@@ -78,6 +78,7 @@ else
 
   pushd ${KFP_MANIFEST_DIR}/dev
   kustomize build . | kubectl apply -f -
+  kubectl delete deployment cache-server -n ${NAMESPACE}
```

Comment:
nit: can you add a TODO comment above, that says `after the release next to 0.3.0, we can remove this line`